src/ILCompiler/repro/Program.cs

```diff
@@ -3,11 +3,540 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Generic;
```

Comment:
It's probably obvious, but please don't check in the changes to the repro project.